fairlearn/datasets/_fetch_acs_income.py

```diff
@@ -86,12 +86,17 @@ def fetch_acs_income(
             Array of ordered feature names used in the dataset.
         DESCR : string
             Description of the ACSIncome dataset.
-
-    (data, target) : tuple of (numpy.ndarray, numpy.ndarray)
-        if ``return_X_y`` is True and ``as_frame`` is False
-
-    (data, target) : tuple of (pandas.DataFrame, pandas.Series)
-        if ``return_X_y`` is True and ``as_frame`` is True
+        categories : dict or None
+            Maps each categorical feature name to a list of values, such that the
+            value encoded as i is ith in the list. If ``as_frame`` is True, this is None.
+        frame : pandas DataFrame
+            Only present when ``as_frame`` is True. DataFrame with ``data`` and ``target``.
+        
+    (data, target) : tuple if ``return_X_y`` is True
```

Comment:
Thanks for making this just like the sklearn fetch_openml API docs!

I'm mildly surprised that they don't mention the data type for data and target (ndarray vs DataFrame/Series). I guess they do mention it above with data and target so it's fine by me like this.

@fairlearn/fairlearn-maintainers any thoughts? If you're fine with this please upvote so that @Tizpaz can move on to the other `fetch_*` functions in the datasets module.

Reply:
@Tizpaz if you prefer to have this PR for ACS Income only that's fine, too! It's entirely up to you...

Reply:
Thanks, @romanlutz. I am fine with doing this to all datasets in `fetch_*`, especially for consistency (we should also be fair to all of our datasets!!).

Reply:
yeah data and target are explained above, I'm happy as is.